SQL.js

```diff
@@ -45,7 +45,7 @@ class SqlStatement {
     return new SqlStatement(result.strings, result.values)
   }
 
-  generateString (type) {
+  generateString (type, namedValueOffset = 0) {
```

Comment:
I'm not sure whether to add `namedValueOffset` to the type definition, as it doesn't have much use externally so should probably be an internal-only param.

I could perhaps rename this to  `_generateString` and make a new function `generateString` that calls `_generateString` with the default namedValueOffset.

Reply:
why do we even have this in the type definition? it's not supposed to be called from client code, right?

Reply:
I'm ok with that